CefSharp.WinForms.Example/Handlers/MenuHandler.cs

```diff
@@ -10,16 +10,24 @@ void IContextMenuHandler.OnBeforeContextMenu(IWebBrowser browserControl, IBrowse
         {
             //To disable the menu then call clear
             // model.Clear();
+
+            bool removed = model.Remove(132); // Remove "View Source" option
```

Comment:
Hey,

Thanks for the quick and very detailed comments/suggestions!

I did some search trying to find the enum before committing my changes, but no luck :/ I do not like inline values either.

Regards,
Zoltan

From: Alex Maitland [mailto:notifications@github.com]
Sent: Tuesday, August 25, 2015 3:54 AM
To: cefsharp/CefSharp <CefSharp@noreply.github.com>
Cc: Zoltan Kovacs <zkovacs@smallfootprint.com>
Subject: Re: [CefSharp] Exposing more methods in IMenuModel to facilitate context menu custom… (#1219)


In CefSharp.WinForms.Example/Handlers/MenuHandler.cs<https://github.com/cefsharp/CefSharp/pull/1219#discussion_r37822725>:

> @@ -10,16 +10,24 @@ void IContextMenuHandler.OnBeforeContextMenu(IWebBrowser browserControl, IBrowse

>          {

>              //To disable the menu then call clear

>              // model.Clear();

> +

> +            bool removed = model.Remove(132); // Remove "View Source" option

I think we probably need to expose the enum for commands, so they're a little more user friendly.

—
Reply to this email directly or view it on GitHub<https://github.com/cefsharp/CefSharp/pull/1219/files#r37822725>.